src/Sylius/Component/Resource/Model/CodeAwareInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Resource\Model;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+interface CodeAwareInterface
```

Comment:
IMO such interface is pointless as it's used in __one__ place, while other interfaces in Resource component are widely used across all components.